### PR TITLE
Add "Secret Sauce" blogpost

### PR DESCRIPTION
**Pull request by SeanTAllen**

### CODE REVIEW

**Comment by SeanTAllen** [comment]
@aturley can you verify that this formats correctly on your machine as well as mine (you'll need to be set up with hugo) and that I'd didn't mess anything up. Final "sanity" check. 

It looks good to me, but another set of eyes would be good.

File: content/post/secret-sauce.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ 
+Want to make a streaming data application go slow? Add a bunch of calls to a remote data store. Those calls are going to end up being a bottleneck. To maximize performance, you need to keep your data and computation close to each other.
+ 
+Imagine an application that tracks the price activity of stocks. We want to be able to update the state of each stock as fast as possible. We have at our disposal three sixteen-core computers. We want to put all forty-eight cores to work updating price information. To achieve this, we need to be able to update each stock independently. 
+ 
+Wallaroo's state object API provides independent, parallelizable individual state “entities.”  In our application, each state object would be the data for a given stock symbol. In ["Life Beyond Distributed Transactions"](https://blog.acolyer.org/2014/11/20/life-beyond-distributed-transactions/)., Pat Helland presents a means of avoiding data coordination. Wallaroo's "state objects" closely resemble the independent "entities" that Pat discusses as being key to scaling distributed data systems. The state object API makes it easy to partition state to avoid coordination among partitions.
```
Comment (aturley) [inline content/post/secret-sauce.md]: The period after the name of the article should not be there.

> In ["Life Beyond Distributed Transactions"](https://blog.acolyer.org/2014/11/20/life-beyond-distributed-transactions/)., Pat Helland presents a means of avoiding data coordination.

**Review by aturley — APPROVED** [review]
Fix that period and I think you're good.

File: content/post/secret-sauce.md (hunk 1 above)
Comment (SeanTAllen) [reply content/post/secret-sauce.md]: Removed and force pushed over.

**Comment by SeanTAllen** [comment]
I just rebased this with the fix on source, it should be good and now pass CI.

**Comment by aturley** [comment]
I approve of this. Publish when ready.

**Pull request merged**